config/locales/helpers/fr.yml

```diff
@@ -4,231 +4,230 @@ fr:
       organization:
         name: Acme Inc.
         slug: acmeinc
-        url: https://yoursite.com
-        location: Halifax, Nova Scotia
+        url: https://votresite.com
+        location: France, Toulouse
         email: mail@example.com
         company_size: "..."
         story: "..."
         tech_stack: "..."
         cta_body_markdown: "..."
-        cta_button_text: Visit Acme Site
-        cta_button_url: https://example.com
+        cta_button_text: Visiter le site Acme 
+        cta_button_url: https://votresite.com
         proof: "..."
       response_template:
         title: "..."
       user:
         name: John Doe
         email: john.doe@example.com
         username: johndoe
-        feed_url: https://yoursite.com/feed
+        feed_url: https://votresite.com/feed
     application_helper:
-      app_logo: App logo
+      app_logo: Logo de l'application
       follow:
         aria_label:
-          Organization: 'Follow organization: %{name}'
-          Tag: 'Follow tag: %{name}'
-          User: 'Follow user: %{name}'
-          default: 'Follow %{type}: %{name}'
+          Organization: "Suivre l'organisation: %{name}"
+          Tag: 'Suivre le tag: %{name}'
+          User: "Suivre L'utilisateur : %{name}"
+          default: "Suivre %{type}: %{name}"
         text:
           Organization: Suivre
           Tag: Suivre
           User: Suivre
           default: Suivre
-      required: Required
+      required: Requis
       subtitle:
-        week: Top posts this week
-        month: Top posts this month
-        year: Top posts this year
-        infinity: All posts
-        latest: Latest posts
+        week: Principaux messages de la semaine
+        month: Principaux messages du mois
+        year: Les meilleurs messages de l'année
+        infinity: Tous les messages
+        latest: Derniers messages
       title_text: "%{title} - %{timeframe}"
     articles_helper:
       logo: "%{service_name} logo"
-      most_comments: Most Comments
-      most_reactions: Most Reactions
-      most_views: Most Views
-      recently_created: Recently Created
-      recently_published: Recently Published
-    authentication_helper:
-      any_of_those: any of those
-      email_password: Email & Password
-      reminder: 'Reminder: you used %{method} to authenticate your account, so please use %{dem} to sign in if prompted.'
-      that: that
-      invite_only: You cannot do this until you disable Invite Only Mode
+      most_comments: Le plus de commentaires
+      most_reactions: Le plus de réactions
+      most_views: Le plus de vues
+      recently_created: Récemment créé
+      récemment_publié: Récemment publié
+    authentification_helper:
+      any_of_those: n'importe lequel de ces éléments
+      email_password: Email et mot de passe
+      reminder:  'Rappel : vous avez utilisé %{method} pour authentifier votre compte, veuillez donc utiliser %{dem} pour vous connecter si vous y êtes invité'
+      that: que
```

Comment:
check: "that" can either be used to qualify (_que_) or demonstrate (_ça / cela_) - check where we use this translation key for the right context.

Oh, it doesn't matter, looks like this key is used nowhere?

```
app/helpers/articles_helper.rb:4:      [I18n.t("helpers.articles_helper.recently_created"), "creation-desc"],
app/helpers/articles_helper.rb:5:      [I18n.t("helpers.articles_helper.recently_published"), "published-desc"],
app/helpers/articles_helper.rb:6:      [I18n.t("helpers.articles_helper.most_views"), "views-desc"],
app/helpers/articles_helper.rb:7:      [I18n.t("helpers.articles_helper.most_reactions"), "reactions-desc"],
app/helpers/articles_helper.rb:8:      [I18n.t("helpers.articles_helper.most_comments"), "comments-desc"],
app/helpers/articles_helper.rb:31:        title: I18n.t("helpers.articles_helper.logo", service_name: service_name),
```

Reply:
I leave that for the moment like this?

Reply:
yes, from what I can see nothing to do here on your side.

- it doesn't matter how this "that" was interpreted since it's not used from what I see
- removing this is out of scope of the work you're doing here, and it's better to focus on one thing at a time

Reply:
FYI, the key is called here, as a demonstrative opposed to "any of those"; both to be injected as a part of the text of the key `reminder`.

https://github.com/forem/forem/blob/07fff2f8efe997477706d75a95ef0d25c12dc48e/app/helpers/authentication_helper.rb#L30

I'm not sure whether this portion of code is further called on or not, though.

Reply:
@yheuhtozr thanks for pointing this out, if it's a demonstrative then _ce_ rather than _que_ seems most appropriate here.